src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+  padding-top: 20px;
+}
+
+form {
+  width: 100%;
+}
+
+form:nth-of-type(1) {
+  margin-bottom: 20px;
+}
+
+.input-big {
+  width: 100%;
+  height: 70px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  padding-left: 62px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+}

```

Comment:
And general advise - take a look over 2 point of [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) - there are a lot of repetitive styles with `.input-small` - try to create common classes which you will use for both small and big elements, and additional classes for `small` - where you will override differences